### PR TITLE
Feature/spinner

### PR DESCRIPTION
**Pull request by marcoalfonso**

### CODE REVIEW

File: app/components/LoadingButton/LoadingButton.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
+/**
+ * LoadingButton.js
+ *
+ * Wraps the loading indicator in a tag with the .btn--loading class
+ */
+
+import React from 'react'
+import LoadingIndicator from '../../components/LoadingIndicator/LoadingIndicator'
+
+function LoadingButton(props) {
```
Comment (Blackfry) [inline app/components/LoadingButton/LoadingButton.js]: are we doing es6 per FE meeting?

Comment (matt-sm) [reply app/components/LoadingButton/LoadingButton.js]: We should wait until we finalise the Airbnb style rules we will follow

File: app/reducers/appReducer.js [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    case SENDING_REQUEST:
+      return {
+        ...state,
+        currentlySending: action.sending
+      }
+
+    case SET_ERROR_MESSAGE:
+      return {
+        ...state,
+        message: action.message
```
Comment (matt-sm) [inline app/reducers/appReducer.js]: You need to change `ErrorBox` cause it's still reading from user.message

Should we also change this to `action.errorMessage`?

**Comment by matt-sm** [comment]
Maybe beyond scope of this, but what about Forms that need to call api to pre-load data?

**Pull request merged**